docs/migration-guide/overview.mdx

```diff
@@ -21,14 +21,90 @@ All breaking changes are listed below. If you encounter changes that are not exp
 
 ## Installation
 
-Payload 3.0 requires a set of auto-generated files that you will need to bring into your existing project. The easiest way of aquiring these is by initializing a new project via `create-payload-app`, then replace the provided Payload Config with your own.
+Payload 3.0 requires a set of auto-generated files that you will need to bring into your existing project. The easiest way of acquiring these is by initializing a new project via `create-payload-app`, then replace the provided Payload Config with your own.
 
 ```bash
   npx create-payload-app@beta
 ```
 
 For more details, see the [Documentation](https://payloadcms.com/docs/getting-started/installation).
 
+1. **Install new dependencies of payload, next.js and react**:
+
+    ```bash
+      pnpm i next react react-dom payload @payloadcms/ui @payloadcms/next
+    ```
+
+    Also install the other @payloadcms packages specific to the plugins and adapters you are using. Depending on your project, these may include:
+      - @payloadcms/db-mongodb
+      - @payloadcms/db-postgres
+      - @payloadcms/richtext-slate
+      - @payloadcms/richtext-lexical
+      - @payloadcms/plugin-form-builder
```

Comment:
Many of these plugin don't need to be listed here bc they have not changed from 2.0->3.0